test/alternator/test_tracing.py

```diff
@@ -70,6 +71,45 @@ def with_tracing(dynamodb):
     if response.status_code != 200 or response.content.decode('utf-8') != '0':
         pytest.skip('Failed to verify tracing disabled')
 
+# Similarly to the fixture above, slow query logging is enabled only for the run of the
+# test function.
```

Comment:
Would be nice to write here what you this fixture sets the "slow query" threshold to be. I see below "1". Is that 1us? 1ms? Or what?

Reply:
Oh, it was supposed to be 0 actually, but 1 is equally effective. It's in microseconds, I'll add a small comment explaining what happens.